docs/spec/lcd/api.md

```diff
@@ -0,0 +1,847 @@
+| new_password    | string | null    | true     | password before |
+| repeat_password | string | null    | true     | password before |
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+     "updated":name
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "cannot update the corresponding key",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+### /keys/{name} - DELETE
+
+url: /keys/{name}, Method: DELETE
+
+Functionality: Delete key from keystore
+
+Parameters:
+
+| Parameter | Type   | Default | Required | Description      |
+| --------- | ------ | ------- | -------- | ---------------- |
+| password  | string | null    | true     | password of keys |
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+     "deleted":name
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "cannot delete the corresponding key",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+## ICS20 - TokenAPI
+
+The TokenAPI exposes all functionality needed to query account balances and send transactions.
+
+### /balance/{account} - GET
+
+url: /balance/{account}, Method: GET
+
+Functionality:
+
+Parameters:
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+     {
+         "atom": 1000,
+         "photon": 500,
+         "ether": 20
+     }
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "Invalid account",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+### /create_transfer - POST
+
+url: /create_transfer, Method: **POST**
+
+Functionality: transfer asset
+
+Parameters:
+
+| Parameter | Type   | Default | Required | Description                 |
+| --------- | ------ | ------- | -------- | --------------------------- |
+| from  | string | null    | true     | address from                |
+| to  | string | null    | true     | address want to send to     |
+| amount  | int    | null    | true     | amount of the token         |
+| denomonation  | string | null    | true     | denomonation of the token   |
+
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+     ""transaction": "[]bytes of a valid transaction bytes to be signed for that zone"
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "Insufficient Funds",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+### /signed_transfer - POST
+
+url: /signed_transfer, Method: POST
```

Comment:
More REST-ful alternative: POST `/txs`

Reply:
This way we also do broadcast all transaction against one endpoint instead of creating a different endpoint for each signed transaction broadcast.